### ISSUE TITLE
Questions about flushing and babel plugin

### ISSUE THREAD

**Issue by wmertens** [faceyspacey/webpack-flush-chunks#77]
## Question 1
Is the babel plugin required? Or is it only helping for `import()` statements with calculated values?

## Question 2
When I do SSR, I do 2 render passes: the first one to request data, and the second one after all data was finalized.
Only the components that were used in the second render pass should be preloaded on the client

However, I don't see a way to reset the cached list of required chunks. How does this work?

## Question 3
React support streaming rendering, but that means that multiple renders might be running at the same time. How does the plugin know what chunks are required in what tree?

Basically, I am surprised there's no `<ChunkProvider>` around the React tree that is used to mark the used imports…
